### DISCUSSION TITLE
Advice on scanning through frequencies for hard to find meters

### DISCUSSION THREAD

**Discussion by JoeDaddy7105** [bemasher/rtlamr#184] [Q&A] [answered]
I have a gas meter that I am having trouble finding. It is an Itron 40G, ERT type 2:
![PXL_20210317_224355404](https://user-images.githubusercontent.com/24470441/111574733-a76ea580-8772-11eb-80e7-c5b9b6e3a95b.jpg)

I have tried scanning each msgtype separately for several minutes as well as using all. I see on the compatible meters list the suggestion to scan between 902-928 MHz which I have tried in 1MHz increments but I am wondering if I need to go more granular like 0.1MHz increments. Is there a way to run rtlamr with a range of frequencies. Something like 902-928MHz in 0.1MHz increments and run each for 2mins? Or do you have any guidance on best practices for how granular to increment the frequencies, how long to scan each frequency and if we should only do one protocol at a time? What other flags should I try incrementing (symbollength, etc.)? Does the frequency changing only work for r900 or is it for all protocols?

This might help me search for my water meter too which is in a pit and not very accessible but I am thinking if I can search for just ERT type 11 or 13 I can narrow it down and then watch for consumption changes as I turn on and off some faucets. I am mostly only finding ERT type 5 & 7

**Comment by bemasher** [comment, marked as answer]
Listening for only a few minutes at a time is probably causing you to miss the transmissions. I previously had a 40G that transmitted only when woken up, and my new meter only transmits every ~15 minutes. It is unlikely you will need to scan around to find where it is transmitting since nearly all ERT compatible meters use FHSS, you will eventually catch messages if you park on a specific channel.

Your meter is fairly old, so I would suggest parking on a particular frequency within the band it uses, and listen for several hours or days to see if it is transmitting.

**Reply by JoeDaddy7105** [reply]
Thanks @bemasher! Would it be better to set msgtype to all or focus on a specific protocol like SCM which is what I am guessing it would transmit on? It seems like when I scan with all, I don't get as many results as when I focus on a specific protocol so I assume I would have better luck to lock in on one protocol and continuously scan for hours until something gets picked up.

Also, as for parking on a specific frequency, is that by setting the centerfreq flag and how would I figure out the band it might be using to help narrow things down?

**Reply by bemasher** [reply]
You'll want to read the FCC test report for your meter, you should be able to figure out the frequencies it uses from there. I'd also recommend using `-msgtype=all` if you aren't sure what messages your meter transmits, but your meter is likely transmitting SCM. If the system you're running rtlamr on isn't powerful enough to run `-msgtype=all`, then `-msgtype=scm` should be fine.

**Comment by sofakng** [comment]
Did you ever find the setting/frequency for this meter? I have the exact same meter (gas) and I'd also like to read it but can't seem to find it.
